sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/WebJobsEventHubTestBase.cs

```diff
@@ -73,20 +75,19 @@ protected void ConfigureTestEventHub(IHostBuilder builder)
                         {"AzureWebJobsStorage", StorageTestEnvironment.Instance.StorageConnectionString}
                     });
                 })
-                .ConfigureServices(services =>
-                {
-                    services.Configure<EventHubOptions>(options =>
-                    {
-                        options.CheckpointContainer = Guid.NewGuid().ToString("D").Substring(0, 13);
-                        // Speedup shutdown
-                        options.EventProcessorOptions.MaximumWaitTime = TimeSpan.FromSeconds(5);
-                    });
-                })
                 .ConfigureDefaultTestHost<T>(b =>
                 {
                     b.AddEventHubs(options =>
                     {
                         options.EventProcessorOptions.TrackLastEnqueuedEventProperties = true;
+                        options.EventProcessorOptions.MaximumWaitTime = TimeSpan.FromSeconds(5);
+                        if (initialOffsetOptions != null)
+                        {
+                            options.InitialOffsetOptions = initialOffsetOptions;
+                        }
+                        options.CheckpointContainer = _checkpointContainerName;
+                        // We want to validate the default options configuration logic for setting initial offset and not implemente it here
+                        EventHubWebJobsBuilderExtensions.ConfigureOptions(options);
```

Comment:
The default logic should always run, this callback is for additional changes.

Reply:
Are you referring to the comment? This was copy pasted. Will remove.